cmd/samples/fileprocessing/workflow.go

```diff
@@ -30,13 +30,12 @@ func init() {
 
 //SampleFileProcessingWorkflow workflow decider
 func SampleFileProcessingWorkflow(ctx workflow.Context, fileID string) (err error) {
-	// step 1: download resource file
 	// step 1: download resource file
 	ao := workflow.ActivityOptions{
 		ScheduleToStartTimeout: time.Second * 5,
 		StartToCloseTimeout:    time.Minute,
 		HeartbeatTimeout:       time.Second * 2, // such a short timeout to make sample fail over very fast
-		RetryPolicy:            &cadence.RetryPolicy{
+		RetryPolicy: &cadence.RetryPolicy{
```

Comment:
did you run go fmt over this code?

Reply:
I just checked. This is the output of go fmt. Not sure why the original code has a different format.